sem_covid/entrypoints/notebooks/topic_modeling/topic_modeling_wrangling/lda_modeling.py

```diff
@@ -0,0 +1,24 @@
+
+import pandas as pd
+from gensim.models import LdaMulticore, TfidfModel
+import gensim.corpora as corpora
+
+
+class WordsModeling(object):
```

Comment:
this could be a function

Reply:
I thought about that, but both functions uses the same arguments and thought it will be a good idea to create a class